Rn okta364354 remove dev console (#1905)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app/configure-packages/index.md

### Configure the SDK

> You need the following values from the Okta Application and the Admin Console that you worked with in <GuideLink link="../create-okta-application">Create an okta application</GuideLink>:

Reviewer: ```suggestion
You need the following values from the Okta Application and the Admin Console that you worked with in <GuideLink link="../create-okta-application">Create an Okta application</GuideLink>:
```

* **Client ID** &mdash; Find it in the applications list or on the **General** tab of your app integration.

> * **Okta Domain** &mdash; The Okta Domain can be found on the Admin Console's global header in the upper-right corner. Click the section which displays your email and company name.  A drop-down menu will appear and display general org information including the full Okta domain (e.g. subdomain.okta.com).

Reviewer: ```suggestion
* **Okta Domain** &mdash; You can find the Okta Domain in the Admin Console's global header in the upper-right corner of the page. Click the section that displays your email and company name.  A drop-down box appears and displays general org information including the full Okta domain (for example, subdomain.okta.com).
```
